apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -7,20 +30,44 @@ export interface MultilingualTextPresenterProps {
 export const MultilingualTextPresenter = ({
     text,
 }: MultilingualTextPresenterProps): JSX.Element => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
     const { items } = text;
 
-    if (!Array.isArray(items)) {
-        throw new Error(`invalid input to Multlingual text!: ${text}`);
-    }
+    const textItemWithDefaultLanguage =
+        items.find((item) => item.languageCode === defaultLanguageCode) || null;
+
+    const translations = items.filter((items) => items.languageCode !== defaultLanguageCode);
 
     return (
-        <>
-            {/* TODO We need to separate the original in some way */}
-            {items.map(({ languageCode, text, role }) => (
-                <div key={`${languageCode}-${role}`}>
-                    {`{${languageCode}} [${role}]`} {text}
-                </div>
-            ))}
-        </>
+        <Box>
+            <Accordion defaultExpanded variant="outlined">
+                <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                    <Typography variant="h3" sx={{ margin: '0' }}>
+                        {isNullOrUndefined(textItemWithDefaultLanguage)
+                            ? 'Translations'
+                            : textItemWithDefaultLanguage.text}
+                    </Typography>
+                    <Tooltip
+                        title={`${getLabelForLanguage(
+                            textItemWithDefaultLanguage.languageCode
+                        )}, '${textItemWithDefaultLanguage.role}'`}
+                    >
+                        <IconButton>
```

Comment:
I like the language button and tooltip, but I'm finding it a bit busy, especially in Terms where the resource icon is also a globe.  What do you guys think?